src/sumo_store.erl

```diff
@@ -167,7 +167,7 @@ start_link(Name, Module, Options) ->
     {workers, PoolSize},
     {worker, {?MODULE, [Module, Options]}}
   ],
-  wpool:start_pool(Name, WPoolConfigOpts ++ WPoolOptions).
+  wpool:start_pool(Name, lists:ukeysort(1, WPoolConfigOpts ++ WPoolOptions)).
```

Comment:
How will this work when you have the same key with different values in both lists? The idea is that the one in `WPoolConfigOpts` should override the one in `WPoolOptions`, I believe.

Reply:
yes this removes any duplicates and only takes the first occurence of key in `[{key,value}]`.

```
1> lists:ukeysort(1,[{test, 1}, {test,2}]).
[{test,1}]
2> lists:ukeysort(1,[{test, 1}, {test,2}]++[{test,3},{test,4}]).
[{test,1}]
3> lists:ukeysort(1,[{test, 2}, {test,1}]++[{test,3},{test,4}]).
[{test,2}]
4> lists:ukeysort(1,[{test, 2}, {test,1}]++[{test,3},{test,4},{other,2}]).
[{other,2},{test,2}]
```